src/app/web-app.component.ts

```diff
@@ -113,6 +126,141 @@ export class WebAppComponent implements OnInit {
+        },
+
+        // Collection Sheet
+        'Control.Shift.o': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Create Client
+        'Control.Shift.c': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Create Group
+        'Control.Shift.g': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Create Center
+        'Control.Shift.q': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Frequent Postings
+        'Control.Shift.f': (event: KeyboardEvent): void => {
+          this.router.navigate(['accounting/journal-entries/frequent-postings']);
+          event.preventDefault();
+        },
+
+        // Closure Entries
+        'Control.Shift.e': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Journal Entries
+        'Control.Shift.j': (event: KeyboardEvent): void => {
+          this.router.navigate(['accounting/journal-entries']);
+          event.preventDefault();
+        },
+
+        // Reports
+        'Control.Shift.r': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Accounting
+        'Control.Shift.a': (event: KeyboardEvent): void => {
+          this.router.navigate(['accounting']);
+          event.preventDefault();
+        },
+
+        // Save/Submit Forms
+        'Control.s': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Run Report
+        'Control.r': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Cancel
+        'Control.Shift.x': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Logout
+        'Control.Shift.l': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Search
```

Comment:
This shortcut is for help and not search, also if you see the community app this will take the user to the documentation manual and not the keyboard shortcut page.

Reply:
In the community-app, all the urls have been hardcoded to route according to page you're currently at,  do i need to do the same?

Reply:
You could try and leverage angular's routing to implement this in a cleaner manner. If you are not able to do it for now please mention a relevant TODO comment here.